packages/components/src/components/popover/popover.jsx

```diff
@@ -52,60 +41,85 @@ class Popover extends React.PureComponent {
 
         const icon_class_name = classNames(classNameTargetIcon, icon);
         return (
-            <div
-                className={classNames('dc-popover', className)}
-                id={id}
-                onMouseEnter={this.toggleOpen}
-                onMouseLeave={this.toggleClose}
+            <TinyPopover
+                isOpen={this.state.is_open}
+                position={alignment}
+                transitionDuration={0.25}
+                padding={margin + 8}
```

Comment:
`+8`?

Reply:
To add 8px to it. Easier this way since you don't need to worry about breaking our css class names convention and account for which direction it is padded (can't pad everything)